pygmt/tests/test_coast.py

```diff
@@ -3,6 +3,7 @@
 """
 import pytest
 from pygmt import Figure
+from pygmt.datasets import load_earth_relief
 from pygmt.exceptions import GMTInvalidInput
```

Comment:
> >  Btw, should we also use the static_earth_relief here?
> I think yes.

The static_earth_relief covers the whole Earth though, do we need to spatially subset it to some island?

```suggestion
from pygmt.exceptions import GMTInvalidInput
from pygmt.helpers.testing import load_static_earth_relief
```

Reply:
The static_earth_relief does not cover the whole Earth (please correct me if I'm wrong @weiji14 ). Moreover, no water areas are currently covered. I'm not sure how we should continue. Adjust the static_earth_relief grid or continue with load_earth_relief?